src/etl/helpers/resource_descriptor_helper_2.py

```diff
@@ -23,21 +34,24 @@ def __init__(self):
 
         # Iterate through this new dictionary and convert page lists to dictionaries.
         # These are keyed by the page name.
-        for entry in self.resource_descriptor_dict.keys():
+        for entry in self.resource_descriptor_dict:
```

Comment:
If entry and .keys() the same? This might effect the UI? @sierra-moxon

Reply:
@oblodgett I tested this and I believe that if you only have one thing after the for when you are iterating on a dictionary you will be getting the keys. If you find otherwise please feel free to make the necessary changes and let me know.